ProteoformSuiteGUI/ProteoformFamilies.cs

```diff
@@ -266,5 +267,46 @@ private void cb_buildAsQuantitative_CheckedChanged(object sender, EventArgs e)
             cb_boldLabel.Enabled = cb_buildAsQuantitative.Checked;
             cb_moreOpacity.Enabled = false; //not fully implemented
         }
+
+        private void btn_inclusion_list_all_families_Click(object sender, EventArgs e)
```

Comment:
Could you explain to me what an inclusion list is and why you're coding it in here?